clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -142,6 +142,13 @@ copies of the binaries and reading the raw bytes for each instruction
 in order to obtain the opcode and full operand information.
 See also \ref sec_drcachesim_core.
 
+drmemtrace analysis tools may use the
+#dynamorio::drmemtrace::decode_cache_t library, which handles the
+heavy-lifting of decoding the trace instructions using either the embedded
```

Comment:
nit: I think this should just be `heavy lifting`

Reply:
Done.